### ISSUE TITLE
SCSS:  Issue with double slash comment 

### ISSUE THREAD

**Issue by giuseppeLeggero** [stylelint/stylelint#7679]
### What minimal example or steps are needed to reproduce the bug?

``` scss
  .radio-button {
    margin-left: -2px;
    width: 240px;
    span {
      font-size: 15px;

      // lorem ipsum blablabla color: var(--s80-list-fields-configuration__row__border)
    }
}
```

### What minimal configuration is needed to reproduce the bug?

``` json 
{
  "plugins": ["stylelint-scss"],
  "rules": {}
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIAESB0AnAhlAlhAtAEYCuALqRAHZLAA61KSAtpugOY6X4A2MAZqXhJ8AJgAOADwDc9RkgDuOKKQAWQkQBYADJJkMUAZzGZqdfYz5VS+AzgBeMIQEYArLvqy5SAPTeAgkiQ3BDoQgBurAAU+DYAHFo8OAbWfDgw3FAG+JCUqWzEWKR4lAD6JegQ8mWEIbDoAJSeKAC+HqTNIAA0IKm8AHKYTHCIMBKDYryoBmAGBl3gVHkIIGYotCATxByUButCANrryQCevNyc1tOz6wC6nZ7r6MS8uwg0rZQd3Tl5AGIhLFIywAVgYqPNYGI5ohVkgjqRTukLns4SBuJhSDBkut7gx4YjzpRLjNXkJ1ujMdiQPQOs0gA)

### Which Stylelint-related dependencies are you using?

``` json 
{
  "stylelint": "latest",
  "stylelint-scss": "latest"
}
```

### What did you expect to happen?

No errors are reported.

### What actually happened?

ERROR
Unknown word (CssSyntaxError)
[7:10-15]

```scss
  .radio-button {
    margin-left: -2px;
    width: 240px;
    span {
      font-size: 15px;

      // lorem ipsum color: var(--s80-list-fields-configuration__row__border)loremipsumcolor 
    }
}
```

**The file will not be modified by appending strings to the end of the line.**
eg: loremipsumcolor

The error seems to occur when there are css declarations in the comment.

**Comment by ybiquitous** [comment]
@giuseppeLeggero Thanks for the report with a reproducible demo. You should specify `customSyntax: 'postcss-scss'` to resolve the error, like [`stylelint-config-recommended-scss`](https://github.com/stylelint-scss/stylelint-config-recommended-scss) does so.

Also, our [Awesome List](https://stylelint.io/awesome-stylelint/) might help you find other packages for SCSS.
